src/js/services/googleMap.js

```diff
@@ -15,7 +15,7 @@ export function getLocation(place) {
       if (!res.length) {
         return reject(new Error(`No result for location ${place}`));
       }
-      resolve(res[0].location);
+      resolve({location: res[0].location, countryCode: res[0].country.short_name});
```

Comment:
Dumb question,  what happens if a user puts a location that doesn't resolve to a country code? would the offer still appear in the offer list?

Reply:
I'm fairly sure the offer still appears, but the location is bogus and there would be no country flag. I think Ricardo did that once (he put emojis I think haha) and we still had the offer in the list